contracts/TablelandTables.sol

```diff
@@ -12,26 +12,30 @@ import "./ITablelandTables.sol";
 import "./ITablelandController.sol";
 
 /**
- * @dev Implementation of {ITablelandTables}.
+ * @dev V1 storage layout.
  */
-contract TablelandTables is
+abstract contract TablelandTablesV1Storage is
     ITablelandTables,
     ERC721AUpgradeable,
     ERC721AQueryableUpgradeable,
     OwnableUpgradeable,
     PausableUpgradeable,
-    ReentrancyGuardUpgradeable,
```

Comment:
see https://forum.openzeppelin.com/t/new-storage-layout-is-incompatible-contextupgradeable-sol-inserted-gap/25371

tl;dr, you can't extend here because this contract adds storage, which conflicts with our own slots. i copied the workaround from the issue above. ugly, but we can remove this when we go to mainnet. ideally, we use the diamond storage layout pattern which doesn't have the problem (as easily).